lib/community/digest.js

```diff
@@ -202,7 +202,7 @@ var formatTime = function (start, end, timezone) {
 
 Digest.formatTime = formatTime // for testing
 
-Digest.prototype.postAttributes = post => {
+Digest.prototype.postAttributes = function (post) {
```

Comment:
this is necessary for Digest.test() to work in the console. otherwise `this === {}`